home-application/pom.xml

```diff
@@ -49,23 +49,6 @@
             <artifactId>home-generated-server</artifactId>
             <version>${revision}</version>
         </dependency>
-        <dependency>
-            <groupId>org.springframework.boot</groupId>
-            <artifactId>spring-boot-devtools</artifactId>
```

Comment:
Do we have it somewhere in another place? Where? Seems to me it is a much preferable place.

Reply:
It was mentioned twice so I left only one.

Reply:
Since home-application is the module that produces an executable jar the dependency `spring-boot-devtools` should be here. Please remove it from the duplicated place and leave it here.

Reply:
In my branch here there's still spring-boot-devtools.